bleep-cli/src/scala/bleep/Main.scala

```diff
@@ -122,6 +122,12 @@ object Main {
 
     val watch = Opts.flag("watch", "start in watch mode", "w").orFalse
 
+    val updateAsScalaSteward = Opts.flag("steward", "Use same upgrade strategy as Scala Steward").orFalse
+
+    val updateWithPrerelease = Opts.flag("prerelease", "Allow upgrading to prerelease version if there is any.").orFalse
+
+    val updateSingleOrgOrModule = Opts.argument[String]("The depenency to update, alternatively only the organization name can be passed").orNone
```

Comment:
This should maybe not be orNone?

Reply:
I think it's fine, that it makes it optional